addons/l10n_it_edi/views/l10n_it_view.xml

```diff
@@ -96,6 +157,7 @@
                     attrs="{'invisible': ['|', ('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund')), ('country_code', '!=', 'IT')]}">
                     <group>
                         <group>
+                            <field name="l10n_it_edi_transaction" groups="base.group_no_one" readonly="1"/>
```

Comment:
```suggestion
                            <field name="l10n_it_edi_transaction" groups="base.group_no_one" readonly="1"
                                      attrs:"{'invisible': [('country_code', '!=', 'IT')]}"/>
```

Reply:
Not needed, it's included in a tab that already has that condition
![image](https://user-images.githubusercontent.com/1665365/225665748-77d8be72-be77-44d0-ad0c-b87fd590a55e.png)